css-positioning/relative.css

```diff
@@ -0,0 +1,36 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+  position: relative;
+  right: 260px;
+}
+
+#bulbasaur {
+  position: relative;
+  top: 20px;
+}
+
+#charmander {
+  position: relative;
+  top: 40px;
+}
+
+#squirtle {
+  position: relative;
+  top: 80px;
+}
+
+#pikachu {
+  position: relative;
+  top: 160px;
+}
+
+#jigglypuff {
+  position: relative;
+  top: 300px;
+}
```

Comment:
You can remove the `relative` on each of these individual rules, you already applied it to all of these elements using the `.pokemon > img` rule above 🤓